src/main.js

```diff
@@ -65,11 +66,25 @@ const load = loader({
     },
   },
 
+  pulseClient: {
+    requires: ['cfg', 'monitor'],
+    setup: ({cfg, monitor}) => {
+      return new libPulse.Client({
+        namespace: 'taskcluster-auth',
+        monitor,
+        credentials: libPulse.pulseCredentials(cfg.pulse),
```

Comment:
@djmitche, another problem reported by taskcluster: `AssertionError [ERR_ASSERTION]: options.username is required`. Is there a way to inject a env variable here? Especially because config.yml has this field and it is an env variable.

Reply:
Do the tests run for you locally?  The tests should run without requiring actual Pulse credentials.  That error indicates it's looking for Pulse credentials.

Reply:
no, dustin, I was following errors from taskcluster: https://tools.taskcluster.net/groups/KRZPFaGGS_6dWieWp8ujJg/tasks/KRZPFaGGS_6dWieWp8ujJg/runs/0/logs/public%2Flogs%2Flive.log

Reply:
It's probably best to start by getting the tests running for you locally.  The way fakes for testing are set up for taskcluster-lib-pulse is different from how they were set up for pulse-publisher.  You can see a bit of it here -- https://github.com/taskcluster/taskcluster-auth/pull/173/files#diff-5ab3980e63f908d4556c516d8c88bdc3R262

Reply:
@djmitche yes, I saw and corrected that error o/ and now all tests are passing locally.